bin/rucio

```diff
@@ -1670,26 +2282,55 @@ def list_account_usage(args):
 
     """
     client = get_client(args)
-    table = []
-    if args.rse:
-        usage = client.get_local_account_usage(account=args.usage_account, rse=args.rse)
+
+    if cli_config == 'rich':
+        status = Status('Fetching account usage', spinner=CLITheme.SPINNER, spinner_style=CLITheme.SPINNER_STYLE, console=console)
+        with status:
+            if args.rse:
+                usage = client.get_local_account_usage(account=args.usage_account, rse=args.rse)
+            else:
+                usage = client.get_local_account_usage(account=args.usage_account)
+
+            table_data = []
+            for item in usage:
+                remaining = 0 if float(item['bytes_remaining']) < 0 else float(item['bytes_remaining'])
+                table_data.append([item['rse'], sizefmt(item['bytes'], args.human), sizefmt(item['bytes_limit'], args.human), sizefmt(remaining, args.human)])
+            table_data.sort()
+            table1 = generate_table(table_data, headers=['RSE', 'USAGE', 'LIMIT', 'QUOTA LEFT'], col_alignments=['left', 'right', 'right', 'right'])
+
+            table_data = []
+            usage = client.get_global_account_usage(account=args.usage_account)
+            for item in usage:
+                if (args.rse and args.rse in item['rse_expression']) or not args.rse:
+                    remaining = 0 if float(item['bytes_remaining']) < 0 else float(item['bytes_remaining'])
+                    table_data.append([item['rse_expression'], sizefmt(item['bytes'], args.human), sizefmt(item['bytes_limit'], args.human), sizefmt(remaining, args.human)])
+            table_data.sort()
+            table2 = generate_table(table_data, headers=['RSE EXPRESSION', 'USAGE', 'LIMIT', 'QUOTA LEFT'], col_alignments=['left', 'right', 'right', 'right'])
+
+            status.stop()
+            print_output(table1, table2, console=console, no_pager=args.no_pager)
+
     else:
-        usage = client.get_local_account_usage(account=args.usage_account)
-    for item in usage:
-        remaining = 0 if float(item['bytes_remaining']) < 0 else float(item['bytes_remaining'])
-        table.append([item['rse'], sizefmt(item['bytes'], args.human), sizefmt(item['bytes_limit'], args.human), sizefmt(remaining, args.human)])
-    table.sort()
-    print(tabulate(table, tablefmt=tablefmt, headers=['RSE', 'USAGE', 'LIMIT', 'QUOTA LEFT']))
-
-    table = []
-    usage = client.get_global_account_usage(account=args.usage_account)
-    for item in usage:
-        if (args.rse and args.rse in item['rse_expression']) or not args.rse:
+        if args.rse:
```

Comment:
args.rse should default to None, so this logic is redundant, you can just pass args.rse whenever (ref: https://github.com/rucio/rucio/blob/e6097bd2ee4efb1ade8b4e262a28efdda6fb96a2/lib/rucio/client/accountclient.py#L330 )